scripts/stub-package/package.json

```diff
@@ -0,0 +1,12 @@
+{
+  "name": "@vue/compiler-sfc",
+  "version": "3.0.0",
+  "description": "This is a workaround for a peer dependency on @trivago/prettier-plugin-sort-imports which is not properly marked as optional. To be removed once solved: https://github.com/trivago/prettier-plugin-sort-imports/issues/201",
+  "main": "index.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
+  "keywords": [],
+  "author": "",
+  "license": "ISC"
+}
```

Comment:
This is a stub package to "trick" yarn into thinking `@vue/compiler-sfc` is installed.

That package is a perr optional dependency from `@trivago/prettier-plugin-sort-imports`, however, there's no public release where it is properly marked as optional.

Once they publish a new version including this metadata, we can get rid of this file and its reference in package.json